Makefile

```diff
@@ -22,6 +22,8 @@ K_BIN     := $(K_RELEASE)/bin
 K_LIB     := $(K_RELEASE)/lib
 export K_RELEASE
 
+K_BUILD_TYPE := Debug
```

Comment:
If your primary goal is for kompile to be as fast as possible, this should be FastBuild, not Debug.

Reply:
Switched